### ISSUE TITLE
Numbered list incorrectly put in chat

### ISSUE THREAD

**Issue by CyberKiller40** [gamejolt/issue-tracker#610]
Problem: I write in the chat in 3 separate messages: (sending, by pressing enter after each line)
```
1. something
2. something
3. something

```
What gets sent:
```
1. something
1. something
1. something
```

**Comment by YLivay** [comment]
Seems like a bug with Marked lib? Could be related to that double encoding issue we have in inline code blocks hmmm...

**Comment by CyberKiller40** [comment]
Actually this might be the correct behavior with markdown. In each paragraph markdown starts numbered lists from 1, regardless of the number that was used. And since each chat message is a separate paragraph, then this happens.

Lists should be interpreted only if they are prepended by whitespace in each line though.

**Comment by Agrosis** [comment]
Yeah, this is kind of the expected behavior. If you do a multi-line message, numbered lists work well. But I think we might be able to disable it.

**Comment by YLivay** [comment]
If we need to touch the marked lib we're using we're better off just leaving it as is. Close if necessary hah

**Comment by hworld** [comment]
Should we disable it if there is no new lines in the message? Might solve most of the cases. The ideal scenario would be to only do it if there was at least 2 lines that start with numbers.
